hapi/hedera-protobufs/services/consensus_topic_info.proto

```diff
@@ -118,10 +121,10 @@ message ConsensusTopicInfo {
      * These are fees to be assessed for each submit to this topic.
      * <p>
      * Each fee defined in this set SHALL be evaluated for
-     * each message submitted to this topic, and the resultant
-     * total assessed fees SHALL be charged.<br/>
+     *  each message submitted to this topic, and the resultant
+     *  total assessed fees SHALL be charged.<br/>
```

Comment:
Minor:
None of the other proto files indent continuation lines in these field specifications.  For consistency we shouldn't indent here.

Reply:
Fixed